### DISCUSSION TITLE
[bug] Comment workflow. How to include "share with"

### DISCUSSION THREAD

**Discussion by alex-nap** [YetiForceCompany/YetiForceCRM#16021] [How to]
Hello, I have a problem creating email notifications about creating new comments in campaigns. I would like an automatic email to be sent to users who have been mentioned in the "Share with" box when creating a comment in a specific campaign. At the moment the email only reaches the creator of the comment. I can't relate comment creation to the campaign where comments are created and written.
How can I solve?
**I am attaching the configuration of my workflow.**

![image](https://user-images.githubusercontent.com/102237839/166097523-6c0d4c4e-72c1-462f-92b5-f3c37128c31c.png)

**Comment by vovpff** [comment]
This can be done with handler or workflow function what send email. You can check tickets workflow function as example

**Comment by alex-nap** [comment]
I tried, "Send email from template" is set for tickets but it doesn't work. The problem is that it does not retrieve the primary emails of the users selected in the "share with" field. It is as if the comments module does not see the users configured on the campaign module. I can't resolve. Do you have another idea?

**Comment by vovpff** [comment]
Via custom workflow function you can send email from template but to, cc and bcc can be filled with values (user ids) from campaign record. To implement this you need to write some code.

Now I have sane functionality in tickets module. If somebody add comment into ticket, assigned and shared with users receive message with ticket summary and comment content

**Comment by alex-nap** [comment]
Hi vovpff, do you mean that Yetiforce cannot recall the users connected to the campaign today, in the cc and bcc entries? I have to write some code to activate workflow with recipient "user linked to campaign via field share with?"
Too bad I don't know how to write this code. Is there any way to customize this feature?

**Reply by vovpff** [reply]
Really I don't know what about 6.3 but at 6.1 this functionality is missing.
You can charge to YF team for this functionality
